src/lib/storage.ts

```diff
@@ -16,26 +16,30 @@ import { checkPackageLocal, publishPackage, checkPackageRemote, cleanUpLinksRef,
 import { setupUpLinks, updateVersionsHiddenUpLink } from './uplink-util';
 import { mergeVersions } from './metadata-utils';
 import { ErrorCode, normalizeDistTags, validateMetadata, isObject } from './utils';
-import type { IStorage, IProxy, IStorageHandler, ProxyList, StringValue, IGetPackageOptions, ISyncUplinks } from '../../types';
-import type { Versions, Package, Config, MergeTags, Version, DistFile, Callback, Logger, IPluginStorageFilter } from '@verdaccio/types';
-import type { IReadTarball, IUploadTarball } from '@verdaccio/streams';
+import { IStorage, IProxy, IStorageHandler, ProxyList, StringValue, IGetPackageOptions, ISyncUplinks, IPluginFilters } from '../../types';
+import { IReadTarball, IUploadTarball, Versions, Package, Config, MergeTags, Version, DistFile, Callback, Logger } from '@verdaccio/types';
 import { hasProxyTo } from './config-utils';
 import { logger } from '../lib/logger';
+import { GenericBody } from '@verdaccio/types';
+import { VerdaccioError } from '@verdaccio/commons-api';
 
 class Storage implements IStorageHandler {
-  localStorage: IStorage;
-  config: Config;
-  logger: Logger;
-  uplinks: ProxyList;
-  filters: Array<IPluginStorageFilter>;
+  public localStorage: IStorage;
+  public config: Config;
+  public logger: Logger;
+  public uplinks: ProxyList;
+  public filters: IPluginFilters;
```

Comment:
do these really need public scope?

Reply:
Not in many, frankly at some point I did this mechanically, seems a reason for PR.